pkg/agent/containerd/containerd.go

```diff
@@ -115,10 +116,14 @@ func Run(ctx context.Context, cfg *config.Node) error {
 // any .txt files are processed as a list of images that should be pre-pulled from remote registries.
 // If configured, imported images are retagged as being pulled from additional registries.
 func PreloadImages(ctx context.Context, cfg *config.Node) error {
-	fileInfo, err := os.Stat(cfg.Images)
-	if os.IsNotExist(err) {
+	err := os.MkdirAll(cfg.Images, 0744)
+	if err != nil {
+		logrus.Errorf("Unable to create agent/images folder in %s: %v", cfg.Images, err)
```

Comment:
Why are we creating this now? Must the folder exist in order to watch it? Is there any other way to handle this?

Reply:
What happens if the user makes the images dir a symlink to another path? Is that handled properly?